selfhosted/actions.yml

```diff
@@ -0,0 +1,13 @@
+platform: MYPLATFORM
+
+qubits: [0]
+
+format: csv
```

Comment:
This is no longer necessary. It is not generating any errors but it is better to remove it.
I like a lot the fact that you are using qibocal directly to test platform in CI! In the future we could also use something else other than the single shot classification or potentially more than one protocol.